src/bosh-director/spec/unit/instance_updater/state_applier_spec.rb

```diff
@@ -109,6 +112,7 @@ module Bosh::Director
         is_canary: false,
         wait_for_running: true,
         logger: logger,
+        task: task,
       )
     end
```

Comment:
```suggestion
        expect(task).to receive(:advance).with(10, status: 'installing packages')
        expect(task).to receive(:advance).with(10, status: 'configuring jobs')
    end
```

Reply:
I think you probably want to test the positive case but if you have it in another unit test feel free to disregard

Reply:
Wanted to explicitly test it would not fail of no task was provided, the positive case is tested via the `allow` mocks: https://github.com/cloudfoundry/bosh/pull/2266/files/ed0bc2595cdded51f1674394cb3c0aced1fdddaf#diff-d6e100c8f783ef9087e049547b1cb645R102-R103